### PR TITLE
BROKER: Insert Shopping

### PR DESCRIPTION
**Pull request by tharaniannamalai**

### CODE REVIEW

File: ShoppingList.ConsoleApp/Models/Shopping/Shopping.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿// ------------------------------------------------
+// Copyright (c) MumsWhoCode. All rights reserved.
+// ------------------------------------------------
+
+namespace ShoppingList.ConsoleApp.Models.Samples
+{
+    class Shopping
+    {
+        public int Id { get; set; }
+        public string Text { get; set; }
```
Comment (ShriHumrudha) [inline ShoppingList.ConsoleApp/Models/Shopping/Shopping.cs]: You need to choose the attributes for your model that are relevant for your model - ShoppingList.

File: ShoppingList.ConsoleApp/Models/Shopping/Shopping.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
+﻿// ------------------------------------------------
+// Copyright (c) MumsWhoCode. All rights reserved.
+// ------------------------------------------------
+
+namespace ShoppingList.ConsoleApp.Models.Samples
+{
+    class Shopping
```
Comment (ShriHumrudha) [inline ShoppingList.ConsoleApp/Models/Shopping/Shopping.cs]: Your model name should be a noun. may be 'Item'?

File: ShoppingList.ConsoleApp/Brokers/Storages/IStorageBroker.ShoppingItem.cs [hunk 3]
```diff
@@ -0,0 +1,12 @@
+﻿// ------------------------------------------------
+// Copyright (c) MumsWhoCode. All rights reserved.
+// ------------------------------------------------
+using ShoppingList.ConsoleApp.Models.ShoppingItems;
```
Comment (ShriHumrudha) [inline ShoppingList.ConsoleApp/Brokers/Storages/IStorageBroker.ShoppingItem.cs:5]: Need a new line space between the comments and using statement

File: ShoppingList.ConsoleApp/Models/ShoppingItems/ShoppingItem.cs [hunk 4]
```diff
@@ -0,0 +1,15 @@
+﻿// ------------------------------------------------
+// Copyright (c) MumsWhoCode. All rights reserved.
+// ------------------------------------------------
+
+namespace ShoppingList.ConsoleApp.Models.ShoppingItems
+{
+    class ShoppingItem
+    {
+        public int Id { get; set; }
+        public string Text { get; set; }
```
Comment (ShriHumrudha) [inline ShoppingList.ConsoleApp/Models/ShoppingItems/ShoppingItem.cs]: Name would be a more appropriate variable name.
Also just have 3 maximum. We will add more attributed later. So Id + 2 more.

File: ShoppingList.ConsoleApp/Brokers/Storages/StorageBroker.ShoppingItem.cs [hunk 5]
```diff
@@ -0,0 +1,18 @@
+﻿// ------------------------------------------------
+// Copyright (c) MumsWhoCode. All rights reserved.
+// ------------------------------------------------
+using ShoppingList.ConsoleApp.Models.ShoppingItems;
+
+namespace ShoppingList.ConsoleApp.Brokers.Storages
+{
+    internal partial class StorageBroker : IStorageBroker
+    {
+        List<ShoppingItem> ShoppingItems = new List<ShoppingItem>();
```
Comment (ShriHumrudha) [inline ShoppingList.ConsoleApp/Brokers/Storages/StorageBroker.ShoppingItem.cs:12]: You need to have a namespace here.

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: ShoppingList.ConsoleApp/Models/Shopping/Shopping.cs (hunk 1 above)
Comment (tharaniannamalai) [reply ShoppingList.ConsoleApp/Models/Shopping/Shopping.cs]: Resolved all the comments.

**Review by ShriHumrudha — APPROVED** [review]

**Pull request merged**